src/config.py

```diff
@@ -0,0 +1,7 @@
+config = {
+    "player_one": "X",
+    "player_two": "O",
+    "play_in_spanish": "2",
+    "play_with_symbols": "2",
+    "play_again": "Y",
+}
```

Comment:
This is a good start! A few things:

I think it can help with readability if the config is nested, so the reader understands whats's being configured. It also helps with organization as configs grow, because it can be hard to find values in a giant flat object. This could be as simple as nested all of these under `menu_options`.

I saw that at least one of the config solutions for python allows for dot notation. This can be really nice just because it looks cleaner to write `config.menu_options.play_in_spanish` than `config["menu_options"]["play_in_spanish"]`, plus it's less of a pain to type and might even be better supported by autocomplete.

Are there any other details of the app that make sense to extract into configuration?

None of this is blocking or necessarily needs to be addressed in this PR. Just stuff to percolate on.

Reply:
Ah, I see.  Unfortunately, I'm seeing in multiple places while googling that Python does not support nested dictionaries in that way.  It says it actually doesn't support accessing values in dictionaries that way at all, even if they're not nested (https://www.w3schools.com/python/python_dictionaries_access.asp).  I tried it out in my app and it also did not prompt auto-complete and if I use the dots, it doesn't recognize it!  "nested dictionaries attributes can not be used with dot notation".  What I'm seeing is they do have packages if you want to do that, but I don't know if it's worth it to go that route...

I'll merge this as is and then we can re-visit if we think there's a work-around!